app/views/posts/show.html.erb

```diff
@@ -1,2 +1,24 @@
-<h1>Here is post list</h1>
+<div class="post-details">
+    <h3 class="comment-title">
+      Post # <%= @index %>
+      by <%= @user.name %>
+    </h3>
 
+    <div class="counter-wrapper">
+      <p class="comment-count">Comments: <%= @post.comments.count %></p>
+      <p class="like-count">Likes: <%= @post.likes.count %></p>
+    </div>
+
```

Comment:
- [ ] Kindly ensure that it is designed in accordance to the wirefreme ( the number of likes is not properly placed).

<img width="352" alt="Untitled" src="https://github.com/rbd3/Blog-App2/assets/103386183/0855c853-9ff1-47b3-8d91-5e16bae34a19">